tethysext/atcore/services/resource_spatial_manager.py

```diff
@@ -59,6 +76,44 @@ def get_extent_for_project(self, datastore_name, resource_id):
 
         return extent
 
+    def get_resource_extent_wms_url(self, resource):
+        """
+        Get url for map preview image.
+        Returns:
+            str: preview image url.
+        """
+        # Default image url
+        layer_preview_url = None
+        layer_name = f'{self.WORKSPACE}:{self.get_extent_layer_name(resource.id)}'
+        try:
+            extent = self.get_extent_for_project(datastore_name=f'{self.WORKSPACE}_primary_db',
```

Comment:
is there an easy way to get the datastore_name from the app.DATABASE variable in app.py? If I remember well they will always be the same.

Reply:
They do not need to be the same. The datastore name is not the same as the connection information (name of the database). It is an arbitrary name that happens to be the same in our apps.

Reply:
Changed this to use the DATASTORE property that is now part of the class.